src/widgets/image/lv_image.c

```diff
@@ -219,6 +219,7 @@ void lv_image_set_src(lv_obj_t * obj, const void * src)
     img->w        = header.w;
     img->h        = header.h;
     img->cf       = header.cf;
+    img->rotation = header.rotation;
```

Comment:
Same as mentioned earlier: source image rotation meta info is not relevant for `lv_image`